Toolset/palettes/behaviors/revaboutcardbehavior.livecodescript

```diff
@@ -0,0 +1,88 @@
+﻿script "revAboutCardBehavior"
+on preOpenStack
+   local tEdition, tPath
+   
+   lock screen
+   
+   switch revLicenseType()
+      case "commercial"
+         put "Indy Edition" into field "Edition"
+         break
+      case "community"
+         put "Community Edition" into field "Edition"
+         break
+      case "professional"
+         put "Business Edition" into field "Edition"
+         break
+      default
+         put "Commercial Edition" into field "Edition"
+   end switch
```

Comment:
is this meant to have a "communityplus" switch?

Reply:
Hmm... I missed that... It should use `revEnvironmentEditionProperty("name")` instead of the whole switch.

Reply:
Hah, oh yeah